.github/workflows/build-doc.yml

```
@@ -4,33 +4,69 @@ on:
-concurrency:
-  group: "pages"
-  cancel-in-progress: false
 
 jobs:
   test_linux:
-    name: Deploy Docs
+    name: Build Docs
     runs-on: ubuntu-latest
 
     steps:
     - uses: actions/checkout@v2
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)